otx/cli/manager/config_manager.py

```diff
@@ -527,16 +527,23 @@ def _copy_config_files(self, target_dir: Path, file_name: str, dest_dir: Path) -
                     )
 
                     config = MPAConfig.fromfile(str(target_dir / file_name))
-                    # FIXME: In the CLI, there is currently no case for using the ignore label.
-                    # so the workspace's model patches ignore to False.
-                    # FIXME: Segmentation -> ignore=True
-                    if config.get("ignore", None):
-                        config.ignore = False
-                        print("In the CLI, Update ignore to false in model configuration.")
+                    self._patch_cli_configs(config)
                     config.dump(str(dest_dir / file_name))
                 except Exception as exc:
                     raise CliException(f"{self.task_type} requires mmcv-full to be installed.") from exc
             elif file_name.endswith((".yml", ".yaml")):
                 config = OmegaConf.load(str(target_dir / file_name))
                 (dest_dir / file_name).write_text(OmegaConf.to_yaml(config))
             print(f"[*] \t- Updated: {str(dest_dir / file_name)}")
+
+    def _patch_cli_configs(self, config):
+        """Patch for CLI configurations."""
+        if config.get("ignore", None):
+            # FIXME: In the CLI, there is currently no case for using the ignore label.
+            # so the workspace's model patches ignore to False.
+            config.ignore = False
+            print("In the CLI, Update ignore to false in model configuration.")
+        if hasattr(config, "deterministic") and hasattr(self.args, "deterministic"):
```

Comment:
How about adding `deterministic` or `seed` value to config although `config` doesn't have those keys?

Reply:
This function is executed against multiple config files. The seed or deterministic is expected to be specified in one of several config files.
If the values are declared here, the same variable will be declared in multiple files at the same time.
I don't have any other ideas at the moment, but I have modified the train function so that it will work even if there is no `deterministic` or `seed` in the config.